addons/web/static/src/core/autocomplete/autocomplete.js

```diff
@@ -65,6 +65,15 @@ export class AutoComplete extends Component {
         });
     }
 
+    get activeSourceOptionId() {
+        if (!this.isOpened || !this.state.activeSourceOption) {
+            return;
```

Comment:
A getter should have an explicit return value. In this case, it implicitly returns `undefined` but still.
cf. https://eslint.org/docs/latest/rules/getter-return